### PR TITLE
Add shadow to question banner, fix question banner

### LINKED ISSUE #50: Question banner fix: (1) back navigation (Problems Page) should have `hover` animation (2) exam's name doesn't show
(1) the back navigation (Problems Page) should have `hover` animation (turn into shadowing)
 it should have shadow during the mouse hovering on it

(2) exam's name doesn't show
image

### PR DESCRIPTION
**Pull request by edisonhello**
Resolves #50.

### CODE REVIEW

File: src/app/exam/question-banner/question-banner.component.ts [hunk 1]
```diff
@@ -24,17 +24,26 @@ export class QuestionBanner implements OnInit {
   exam: ExamOverview;
   problem: Problem;
 
+  updateExamInformation(student: Student) {
```
Comment (Johnny850807) [inline src/app/exam/question-banner/question-banner.component.ts:27]: rename `updateExamProgressOverview`

**Review by Johnny850807 — CHANGES_REQUESTED** [review]
Review done

File: src/app/exam/question-banner/question-banner.component.ts (hunk 1 above)
Comment (edisonhello) [reply src/app/exam/question-banner/question-banner.component.ts]: Fixed.

**Review by Johnny850807 — APPROVED** [review]
LGTM

**Review by Chaoyuuu — APPROVED** [review]
LGTM

**Pull request merged**